server/controllers/calendarDetailController.test.ts

```diff
@@ -108,14 +107,14 @@ describe('calendar detail middleware', () => {
       }
       processDetail.mockReturnValue(processReturn)
       getCalendarDayMock.mockResolvedValue(calendarDataRestDay)
-      await calendarDetailMiddleware(req, res, nextMock)
+      await new CalendarDetailController().details(req, res, nextMock)
     })
     it('should request the day details', () => {
       expect(getCalendarDayMock).toHaveBeenCalledTimes(1)
     })
     it('should try and sort the details', () => {
       expect(sortByDisplayType).toHaveBeenCalledTimes(1)
-      expect(sortByDisplayType).toHaveBeenCalledWith([{ start: startTime, end: endTime }], 'displayTypeTime')
+      expect(sortByDisplayType).toHaveBeenCalledWith([{ start: startTime, end: endTime }])
```

Comment:
slightly odd this one.  When converted to typescript there was an error in the controller as `sortByDisplayType` only accepts 0-1 arguments but 2 were passed.  The test here even then checked that two arguments were passed in, even though the second would then be ignored 🤷